### DISCUSSION TITLE
jdk 19 building

### DISCUSSION THREAD

**Discussion by tarzanek** [oracle/opengrok#4169] [Development]
Hey guys
tried to build with jdk 19 and ended with:

java: non-transient instance field of a serializable class declared with a non-serializable type

on lots of serialized classes

it's still a warning, so we can decide to ignore it, but I guess someone can have a look (or someone already did how serialization changed in jdk 19 ? )

**Comment by tarzanek** [comment]
ignore, I didn't have properly synced/reloaded the project in Idea upon JDK switching

**Reply by vladak** [reply]
For the record, I experimented with Java 19 a bit and the first issue I ran into was that Maven got broken.
